src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagEntry.cs

```diff
@@ -10,11 +10,16 @@ namespace Microsoft.Health.Dicom.Core.Features.CustomTag
     /// </summary>
     public class CustomTagEntry
```

Comment:
>CustomTagEntry [](start = 17, length = 14)

I think I wrote this comment in Sneha's PR, but this class is currently mutable (as the properties have `public` setters). If this is intended, then we don't really need the ctors. If we want to make it immutable (which is generally preferable), then we can either make it a `struct` or perhaps even a new `record` type.

Reply:
As Sneha and I share same file, I leave this to Sneha, since her code has more functions.